backend/src/main/java/de/neuefische/saildog/utils/MainSailTrimUtils.java

```diff
@@ -0,0 +1,199 @@
+package de.neuefische.saildog.utils;
+
+import de.neuefische.saildog.enums.*;
+import de.neuefische.saildog.model.MainSail;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MainSailTrimUtils {
+
+    public MainSail calculateMainSailTrim(BoatCourse boatCourse, double wind, double wave){
+        if (boatCourse == BoatCourse.CLOSED_HAULED) {
+            if (wind < 7 && wave < 0.5) {
+                return MainSail
+                        .builder()
+                        .mainSailSheet(SheetState.LOOSE)
+                        .traveller(TravellerState.SLIGHTLY_LUV)
+                        .boomVang(BoomVangState.LOOSE)
+                        .mainSailLuff(LuffFootState.SMOOTH)
+                        .mainSailFoot(LuffFootState.SLIGHTLY_CLOSED)
+                        .build();
```

Comment:
In order to get a better overview of the whole class, i would outsource the builder in a separate function if possible.

Reply:
thx for your review! 😊

do you have something like this in mind?

``` java
// ...some previos code...
    if (wind >= 18 && wave < 1.5) {
                return MainSail
                        .builder()
                        .mainSailSheet(SheetState.LOOSE)
                        .traveller(TravellerState.LEE)
                        .boomVang(BoomVangState.MAX_TIGHT)
                        .mainSailLuff(LuffFootState.SMOOTH)
                        .mainSailFoot(LuffFootState.SLIGHTLY_CLOSED)
                        .build();
    }
    if (wind >= 18 && wave >= 1.5) {
                return mainSailBuilder(
                        SheetState.LOOSE, TravellerState.LEE,
                        BoomVangState.MAX_TIGHT, LuffFootState.SMOOTH,
                        LuffFootState.SLIGHTLY_CLOSED);
            }
    }
  return new MainSail();
  }

    private MainSail mainSailBuilder(SheetState sheetState, TravellerState travellerState,
                                     BoomVangState boomVangState, LuffFootState luffState, LuffFootState footState) {
        return MainSail
                .builder()
                .mainSailSheet(sheetState)
                .traveller(travellerState)
                .boomVang(boomVangState)
                .mainSailLuff(luffState)
                .mainSailFoot(footState)
                .build();
    }
}
```

Reply:
yes 👍